addons/mail/static/tests/chat_window/chat_window_tests.js

```diff
@@ -404,6 +404,25 @@ QUnit.test("Close active thread action in chatwindow on ESCAPE", async () => {
     await contains(".o-mail-ChatWindow");
 });
 
+QUnit.test("Exit chatwindow thread rename on ESCAPE without closing chat window", async () => {
```

Comment:
Name suggestion for improved readability and maintainability of test:
_"ESC cancels thread rename"_

Testing that it doesn't close chat window is deduced from asserting that the thread rename is canceled with a `contains()` assertion (no user interaction necessary to re-open the chat window).